src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +611,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(@StringRes int name, String flag) {
+		return flag + " " + getString(name);
```

Comment:
Could we maybe sort by another string? Then we could sort by the name without flags and display with flags.